docs/getting-started/roadmap.md

```diff
@@ -4,27 +4,35 @@ This document defines a high level roadmap for KubeEdge development.
 
 The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-KubeEdge 1.2 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.
+KubeEdge 1.3 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.
 
-## 2020 Q1 Roadmap
-- Support for monitoring at the edge.
-- Support HA for cloudcore
-- Support exec&logs API for edge application
-- Add protobuf support for data exchange format between cloud and edge
-- Finish scalability test and publish report
-- Support managing clusters at edge from cloud (aka. EdgeSite)
-- Enhance performance and reliability of KubeEdge infrastructure.
-- Support ingress at edge.
-- Improve contributor experience by defining project governance policies, release process, membership rules etc.
-- Improve the performance and e2e tests with more metrics and scenarios.
```

Comment:
I'd like these info is kept here instead of being removed.

Reply:
Haven't removed them, just move to future list instead of Q2.